lib/Doctrine/Common/DataFixtures/AbstractFixture.php

```diff
@@ -71,13 +72,27 @@ public function addReference($name, $object)
      *
      * @see Doctrine\Common\DataFixtures\ReferenceRepository::getReference
      *
-     * @param string $name
+     * @param string      $name
+     * @param string|null $class
+     * @psalm-param class-string<T>|null $class
      *
      * @return object
+     * @psalm-return $class is null ? object : T
+     *
+     * @template T of object
      */
-    public function getReference($name)
+    public function getReference($name, $class = null)
```

Comment:
You can use native type declarations for the new parameters, it will be that less work when migrating to native type declarations in 2.0

Reply:
Indeed, I added the native type declaration for every new param.